.teamcity/src/main/kotlin/promotion/BasePublishGradleDistribution.kt

```diff
@@ -66,6 +66,6 @@ fun BuildSteps.buildStep(extraParameters: String, gitUserName: String, gitUserEm
     gradleWrapper {
         name = "Promote"
         tasks = "$prepTask $stepTask"
-        gradleParams = """-PcommitId=%dep.${RelativeId("Check_Stage_${triggerName}_Trigger")}.build.vcs.number% $extraParameters "-PgitUserName=$gitUserName" "-PgitUserEmail=$gitUserEmail" %additional.gradle.parameters% """
+        gradleParams = """-PcommitId=%dep.${RelativeId("Check_Stage_${triggerName}_Trigger")}.build.vcs.number% $extraParameters "-PgitUserName=$gitUserName" "-PgitUserEmail=$gitUserEmail" -Dorg.gradle.internal.plugins.portal.url.override=%gradle.plugins.portal.url% %additional.gradle.parameters%"""
```

Comment:
Can we extract this parameter for overriding the plugin URL to some extension function/String constant?

Reply:
Done, PTAL!